dev-haskell/quickcheck/quickcheck-2.14.2.ebuild

```diff
@@ -36,6 +34,6 @@ src_configure() {
 	haskell-cabal_src_configure \
 		--flag=-base3 \
 		--flag=base4 \
-		$(cabal_flag old-random old-random) \
+		--flag=old-random \
```

Comment:
Instead of forcing a cabal flag we can remove explicit setting of a cabal flag and let automatic resolution pick the correct branch based on already present `random` version.